### PR TITLE
Update Chinese translation to ver. 2021-05-01

### PR DESCRIPTION
**Pull request by oyyq99999**

### CODE REVIEW

File: chinese/wca-guidelines.md [hunk 1]
```diff
@@ -130,6 +118,9 @@ WCA指南用于补充[WCA规则](regulations:top)。关于WCA的更多信息见
 - 9f5+) [澄清] 如果选手可以参赛却没有开始那次尝试（见[规则A3b2](regulations:regulation:A3b2)），结果才记为DNS。如果选手没有资格进行还原（例如在有及格线轮次中），对于那次还原他不会有任何成绩（包括DNS）。
 - 9p2+) [澄清] 成绩指的是决定相应轮次排名的单次或者平均成绩。
 - 9q+) [建议] 任何项目和轮次应该包含至少2个选手。
+- 9t+) [解释] 由于[9t1](regulations:regulain:9t1)和[9t2](regulations:regulain:9t2)的缘故，某些轮次可能看上去不符合[9p1](regulations:regulation:9p1)、[9m1](regulations:regulation:9m1)、[9m2](regulations:regulation:9m2)或[9m3](regulations:regulation:9m3)的要求。然而，为了满足这些规则而做出的改变会影响到无辜的选手，这就是这些结果要保持不变的原因。
```
Comment (hanwu85) [inline chinese/wca-guidelines.md:121]:
```suggestion
- 9t+) [解释] 由于[9t1](regulations:regulation:9t1)和[9t2](regulations:regulation:9t2)的缘故，某些轮次可能看上去不符合[9p1](regulations:regulation:9p1)、[9m1](regulations:regulation:9m1)、[9m2](regulations:regulation:9m2)或[9m3](regulations:regulation:9m3)的要求。然而，为了满足这些规则而做出的改变会影响到无辜的选手，这就是这些结果要保持不变的原因。
```

Comment (hanwu85) [reply chinese/wca-guidelines.md:121]: some typo from original version.

File: chinese/wca-guidelines.md [hunk 2]
```diff
@@ -130,6 +118,9 @@ WCA指南用于补充[WCA规则](regulations:top)。关于WCA的更多信息见
...
+- 9t++) [附加] 如果在比赛过程中出现了这些改变成绩的情况，这条规则也必须被使用。在这种情况下，[9t1](regulations:regulain:9t1)、[9t2](regulations:regulain:9t2)以及[9t3](regulations:regulain:9t3)也应当适用, 由WCA代表决定。
```
Comment (hanwu85) [inline chinese/wca-guidelines.md:122]: same typo from original version.
```suggestion
- 9t++) [附加] 如果在比赛过程中出现了这些改变成绩的情况，这条规则也必须被使用。在这种情况下，[9t1](regulations:regulation:9t1)、[9t2](regulations:regulation:9t2)以及[9t3](regulations:regulation:9t3)也应当适用, 由WCA代表决定。
```

**Comment by oyyq99999** [comment]
OK those typos were from the official version, fixed now.

**Comment by oyyq99999** [comment]
any updates?

**Comment by Ivan-Brigidano** [comment]
We are still waiting for the travis errors to be fixed by he WST. Unfortunately, we can't merge the PR until the errors are fixed :/

**Comment by oyyq99999** [comment]
reopened as #111

**Pull request closed without merging**